examples/jenkins-image-sample.groovy

```diff
@@ -289,14 +289,14 @@ void actualTest() {*/
     
```

Comment:
are we sure lines 275 to 288 still work @akram ?  what are the details on what you've tested ?